### PR TITLE
Feature/Add Item

### LINKED ISSUE #3: [1pt]create add item form
When the user clicks the "Add item" link in the navigation panel they can see a form for adding a new item.

### PR DESCRIPTION
**Pull request by Thermiee**
## Description
In this PR I created Add items form.

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### Milestone requirements:

- [x] When the user clicks the "Add item" link in the navigation panel they can see a form for adding a new item. 
- [x] When a user fills the form it saves to the database. 

Thank you for taking the time to review this pull request. :pray:

### CODE REVIEW

File: src/redux/flights/flights.js [hunk 1]
```diff
@@ -18,7 +18,9 @@ export const get = () => async (dispatch) => {
 };
 
 export const post = (data) => async (dispatch) => {
+  console.log(data);
```
Comment (ZahraArshia) [inline src/redux/flights/flights.js]: Can you please remove this line before merging? It causes a warning

**Review by ZahraArshia — APPROVED** [review]
Hello @Thermiee,
nice job implementing add item feature 👍 
I have tested it, and it's working perfectly fine, we can merge it now

**Review by saied2035 — APPROVED** [review]
Great work you did! 🎉 
tenor

# Status: Approved 🟢 

please check the comments before you merge 😄

File: src/components/AddItem.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  useEffect(() => {
+    if (item.id) {
+      navigate(`/Item_detail/${item.id}`);
+    }
+  }, [item]);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(post({
+      name, image, flight_Number: flightNumber, price, user_id: user.id,
```
Comment (saied2035) [inline src/components/AddItem.js]: Please, change `flight_number` to `flight_number`

File: src/redux/flights/flights.js (hunk 1 above)
Comment (Thermiee) [reply src/redux/flights/flights.js]: alright will do thanks

File: src/components/AddItem.js (hunk 2 above)
Comment (Thermiee) [reply src/components/AddItem.js]: alright thanks

**Comment by saied2035** [comment]
@Thermiee, please check the linter errors

**Comment by Thermiee** [comment]
> 
@saied2035 
I'm confused about what is causing the linter error.
but I'm on it

**Pull request merged**